roles/kong/templates/secret-ee-admin-gui-auth.j2

```diff
@@ -5,4 +5,4 @@ metadata:
   name: tavros-{{ item[0].name }}-ee-admin-gui-auth
   namespace: kong
 data:
-  admin_gui_auth_conf: {{ lookup('template', 'oidc-config-keycloak.j2') | to_json | string | b64encode }}
+  admin_gui_auth_conf: {{ lookup('template', 'oidc-config-keycloak.j2') | from_json | to_json | string | b64encode }}
```

Comment:
@jam01 So these changes that make the tests succeed somehow fail on an actual run.
`TASK [ms3_inc.tavros.kong : Template Instance EE Files] ************************
Sunday 12 December 2021  01:06:47 +0000 (0:00:00.497)       0:07:36.898 ******* 
changed: [localhost] => (item=[{'ee': {'admin_password': 'MzGutp2c', 'creds': 'default', 'enabled': True, 'keycloak': {'client_secret': 'd47031c9-4bbf-565c-bf83-214beb82ba60', 'realm': 'prod'}}, 'hybrid': True, 'ingress_class': 'prod', 'kuma_mesh_name': 'prod', 'name': 'prod-kong', 'pg': {'password': 'N6O5J:zdWm2-Vzv1TUIi'}}, {'name': 'secret-ee-admin-creds'}])
An exception occurred during task execution. To see the full traceback, use -vvv. The error was: ): the JSON object must be str, bytes or bytearray, not dict
failed: [localhost] (item=[{'ee': {'admin_password': 'MzGutp2c', 'creds': 'default', 'enabled': True, 'keycloak': {'client_secret': 'd47031c9-4bbf-565c-bf83-214beb82ba60', 'realm': 'prod'}}, 'hybrid': True, 'ingress_class': 'prod', 'kuma_mesh_name': 'prod', 'name': 'prod-kong', 'pg': {'password': 'N6O5J:zdWm2-Vzv1TUIi'}}, {'name': 'secret-ee-admin-gui-auth'}]) => changed=false 
  ansible_loop_var: item
  item:
  - ee:
      admin_password: MzGutp2c
      creds: default
      enabled: true
      keycloak:
        client_secret: d47031c9-4bbf-565c-bf83-214beb82ba60
        realm: prod
    hybrid: true
    ingress_class: prod
    kuma_mesh_name: prod
    name: prod-kong
    pg:
      password: N6O5J:zdWm2-Vzv1TUIi
  - name: secret-ee-admin-gui-auth
  msg: |-
    AnsibleError: Unexpected templating type error occurred on (apiVersion: v1
    kind: Secret
    type: Opaque
    metadata:
      name: tavros-{{ item[0].name }}-ee-admin-gui-auth
      namespace: kong
    data:
      admin_gui_auth_conf: {{ lookup('template', 'oidc-config-keycloak.j2') | from_json | to_json | string | b64encode }}
    ): the JSON object must be str, bytes or bytearray, not dict
changed: [localhost] => (item=[{'ee': {'admin_password': 'MzGutp2c', 'creds': 'default', 'enabled': True, 'keycloak': {'client_secret': 'd47031c9-4bbf-565c-bf83-214beb82ba60', 'realm': 'prod'}}, 'hybrid': True, 'ingress_class': 'prod', 'kuma_mesh_name': 'prod', 'name': 'prod-kong', 'pg': {'password': 'N6O5J:zdWm2-Vzv1TUIi'}}, {'name': 'secret-ee-admin-gui-session'}])
`

Reply:
I was thinking about this more and is this only because we need to build the new image with the newest Ansible? Because if that is the case then I think we are good to go.